app/controllers/bugs_controller.rb

```diff
@@ -0,0 +1,55 @@
+class BugsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :authenticate_control!
+
+  def index
+    @bugs = Bug.where(game: current_game).order(:created_at)
+  end
+
+  def new
+    @bug = Bug.new
+    @teams = Team.all
+  end
+
+  def create
+    targetId = bug_params['target'].to_i
+    beneficiaryId = bug_params['beneficiary'].to_i
+    new_params = bug_params.except('target', 'beneficiary')
+    @bug = Bug.new(new_params)
+    @bug.target = Team.find(targetId)
+    @bug.beneficiary = Team.find(beneficiaryId)
+    @bug.klass = Message
+    @bug.game = current_game
+    respond_to do |format|
+      if @bug.save
+        format.html { redirect_to bugs_path, notice: 'Bug Successfully Placed' }
+        format.json {render nothing: true, status: :created}
+      else
+        format.html {render :new}
+        format.json { render json: @bug.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
```

Comment:
I suspect this action will need to do the same annoying manual setting of the Target and Beneficiary that I had to write above. That said, we aren't actually editing Bugs now (only updating to mark them as "finished", so maybe it doesn't need to be a thing?